src/main/java/com/playonlinux/ui/impl/qt/common/QMemoryStream.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2015 Markus Ebner
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.playonlinux.ui.impl.qt.common;
+
+import com.trolltech.qt.core.QByteArray;
+import com.trolltech.qt.core.QIODevice;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * An implementation of QIODevice for Java InMemoryStreams.
+ */
+public class QMemoryStream extends QIODevice {
+
+    private InputStream innerStream;
+
+    public QMemoryStream(InputStream innerStream){
+        this.innerStream = innerStream;
+    }
+
+    @Override
+    public int readData(byte[] bytes) {
+        try{
+            return innerStream.read(bytes);
+        }catch(IOException e){
```

Comment:
Can you set your code formatter so that it follows the following style:

try {
    return innerStream.read(bytes);
} catch(IOException e) {
...